server/pbench/bin/pbench-report-status

```diff
@@ -104,7 +104,8 @@ fi
 
 md5=$(md5sum $file | cut -d' ' -f1)
 
-$dir/pbench-gen-json-payload $timestamp $name $doctype $file > ${tmp}/payload
+# drop the -UTC spec from the timestamp: the date parser of ES 1.7 does not like it
```

Comment:
Can you update this comment to talk about all Elasticsearch versions since this does not appear to be specific to Elasticsearch 1.7?

Reply:
Eliminated the 1.7 from the comment in `pbench-report-status`, the commit message, and the PR description above. I hope that's all.